src/lisvap/utils/__init__.py

```diff
@@ -373,32 +373,46 @@ def get_cuts(in_file):
         settings = LisSettings.instance()
         filename = '{}.{}'.format(os.path.splitext(in_file)[0], 'nc')
         nf1 = Dataset(filename, 'r')
+
+        maskmap_attrs = MaskMapMetadata.instance()
+        cell = maskmap_attrs['cell']
+        mask_x = maskmap_attrs['x']
+        mask_y = maskmap_attrs['y']
+
         if 'lon' in nf1.variables.keys():
             x1 = nf1.variables['lon'][0]
             x2 = nf1.variables['lon'][1]
+            # Detect if the x axis is inverted
+            if int(mask_x + cell) != int(x1):
+                x1 = nf1.variables['lon'][-1]
+                x2 = nf1.variables['lon'][-2]
             y1 = nf1.variables['lat'][0]
             y2 = nf1.variables['lat'][1]
+            # Detect if the y axis is inverted
+            if int(mask_y - cell) != int(y1):
+                y1 = nf1.variables['lat'][-1]
+                y2 = nf1.variables['lat'][-2]
         else:
             x1 = nf1.variables['x'][0]
             x2 = nf1.variables['x'][1]
             y1 = nf1.variables['y'][0]
             y2 = nf1.variables['y'][1]
         nf1.close()
 
-        maskmap_attrs = MaskMapMetadata.instance()
-        if maskmap_attrs['cell'] != round(np.abs(x2 - x1), 5) or maskmap_attrs['cell'] != round(np.abs(y2 - y1), 5):
+        round_x = float(str(round(np.abs(x2 - x1), 5)))
```

Comment:
Do we need to wrap round() with str() and then back to float() again?

Actually we should use np.round here. Something like:
`round_x = np.round(np.abs(x2 - x1), 5)`

Can you try?